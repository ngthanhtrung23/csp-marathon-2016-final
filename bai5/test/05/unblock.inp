..1.22
..1..3
001..3
.45553
.4667.
....7.
