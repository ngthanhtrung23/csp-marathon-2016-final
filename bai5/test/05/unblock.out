15
2 L 1
3 U 1
4 D 1
5 L 1
7 U 3
5 R 2
6 R 1
1 D 3
2 L 3
7 U 1
0 R 3
1 U 3
5 L 1
3 D 3
0 R 1

